op-node/flags/flags.go

```diff
@@ -34,6 +34,18 @@ var (
 		Usage:   "Address of L2 Engine JSON-RPC endpoints to use (engine and eth namespace required)",
 		EnvVars: prefixEnvVars("L2_ENGINE_RPC"),
 	}
+	L2RpcTimeout = &cli.DurationFlag{
```

Comment:
Is this L2 specific or just a RPC / Batch RPC timeout?

Reply:
This option is passed to the `BaseRPCClient`, so I believe it acts as a timeout for both RPC and Batch RPC.